simapp/v2/simdv2/cmd/commands.go

```
@@ -20,6 +21,7 @@ import (
 	confixcmd "cosmossdk.io/tools/confix/cmd"
 
 	"github.com/cosmos/cosmos-sdk/client"
+	sdkclient "github.com/cosmos/cosmos-sdk/client"
```

golangci-lint (1.62.2)

ST1019(related information): other import of "github.com/cosmos/cosmos-sdk/client"

(stylecheck)